node_modules/oae-core/signin/signin.html

```diff
@@ -0,0 +1,80 @@
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="__MSG__CLOSE__">
+                    <span aria-hidden="true">&#215;</span>
+                </button>
+                <h3 id="signin-modal-title">__MSG__SIGN_IN__</h3>
+            </div>
+            <div class="modal-body">
+                <div id="signin-options-container"><!-- --></div>
+            </div>
+            <div id="signin-modal-footer-container"><!-- --></div>
+        </div>
+    </div>
+</div>
+
+<div id="signin-options-template"><!--
+    {if authStrategyInfo.hasExternalAuth}
+        <div id="signin-options-external">
+            <ul class="nav">
+                {var opts = new Object()}
+                ${opts.data = new Object()|eat}
+                ${opts.data.redirectUrl = redirectUrl|eat}
+                ${opts.data.invitationToken = invitationInfo.token|eat}
+                {for strategy in authStrategyInfo.enabledExternalStrategies}
+                    <li>
+                        ${authExternalButton(strategy_index, strategy, opts)}
+                    </li>
+                {/for}
+            </ul>
+        </div>
+        {if authStrategyInfo.hasLocalAuth}
+            <div id="signin-options-divider">
+                <div class="hr">
+                    <span>__MSG__OR__</span>
+                </div>
+            </div>
+        {/if}
+    {/if}
+    {if authStrategyInfo.hasLocalAuth}
+        <div id="signin-options-local">
+            <form id="signin-local-form" role="form">
+                <div class="form-group">
+                    <label for="signin-local-username" class="control-label">__MSG__USERNAME__</label>
+                    <input id="signin-local-username" name="username" type="text" autocapitalize="off" autocorrect="off" class="form-control required" maxlength="255" placeholder="__MSG__ENTER_YOUR_USERNAME__" />
+                </div>
+                <div class="form-group">
+                    <label for="signin-local-password" class="control-label">__MSG__PASSWORD__</label>
+                    <input id="signin-local-password" name="password" type="password" class="form-control required" maxlength="255" placeholder="&#8226;&#8226;&#8226;&#8226;&#8226;&#8226;&#8226;" />
+                </div>
+                <div class="form-group">
+                    <button type="submit" class="btn btn-primary">__MSG__SIGN_IN__</button>
+                </div>
+            </form>
+        </div>
+    {/if}
+    </ul>
+--></div>
+
+<div id="signin-modal-footer-template"><!--
+    <div class="modal-footer">
+        <div id="signin-modal-cancel">
+            <button class="btn btn-link" data-dismiss="modal">__MSG__CANCEL__</button>
+        </div>
+        {if showSignUp}
+            <div id="signin-modal-signup">
+                <span>__MSG__NO_ACCOUNT_YET__</span>
+                <a href="/signup" class="btn btn-link">__MSG__SIGN_UP__</button>
```

Comment:
This is an anchor tag that's closed as a button, which is causing some undesired whitespace below the button:

<img width="1719" alt="screen shot 2015-09-11 at 09 14 11" src="https://cloud.githubusercontent.com/assets/109850/9819902/7e463584-5865-11e5-8837-add0feb4e59d.png">